src/transformers/modeling_utils.py

```diff
@@ -766,17 +783,19 @@ def _get_resized_lm_head(
 
     def init_weights(self):
         """
-        Initializes and prunes weights if needed.
+        Maybe initializes and prunes weights if needed.
```

Comment:
```suggestion
        If needed prunes and maybe initializes weights.
```

Disambiguate what "maybe" refers to